zipline-gradle-plugin/src/test/kotlin/app/cash/zipline/gradle/ZiplineGradlePluginTest.kt

```diff
@@ -519,6 +519,7 @@ class ZiplineGradlePluginTest {
   ): GradleRunner {
     val gradleRoot = projectDir.resolve("gradle").also { it.mkdir() }
     File("../gradle/wrapper").copyRecursively(gradleRoot.resolve("wrapper"), true)
+    File(projectDir, "kotlin-js-store/yarn.lock").delete()
```

Comment:
Sneaking this in. My test was failing locally because I had an old yarn.lock file that the new dependencies didn’t like.

Reply:
I think the other tests have configuration in their build.gradle to always overwrite the file which prevents this problem.